components/broadcast/index.js

```diff
@@ -20,6 +23,12 @@ AFRAME.registerSystem('broadcast', {
       info('Connected', url);
     });
 
+    this.sendQueue = [];
+    this.helperQuaternionReceive = new THREE.Quaternion();
```

Comment:
if multiple components are updating rotation will this break because of race conditions depending on the same helper quaternion?

Reply:
Js is single threaded so shouldn’t?